### ISSUE TITLE
Access token should be automatically refreshed on expiration

### ISSUE THREAD

**Issue by wilzbach** [storyscript/cli#122]
I just got this error:

```
> asyncy login
You're logged in as sebi.wilzbach@gmail.com
> asyncy apps create
Creating application... |
Error: Invalid or expired token
```

The access token used here:

https://github.com/asyncy/cli/blob/db949fe64b79152741b71f357e60158a238c602c/cli/api.py#L32

is never refreshed. At the very least `login` should check whether the current token is valid.

**Comment by judepereira** [comment]
@wilzbach To unblock you, for now I've kicked forward the validity of your token.

**Comment by judepereira** [comment]
Closed with #285
